configuration/items.md

```diff
@@ -202,17 +202,34 @@ Two naming schemes are established in the community for Group names:
 {: #label}
 ### Label
 
-Label text is used to describe an Item in a human-readable way.
-Graphical UIs will display the label text when the Item is included, e.g. in [Basic UI]({{base}}/addons/uis/basic/readme.html) in a [Sitemap]({{base}}/configuration/sitemaps.html) definition.
+The label is used to describe an Item in a human-readable way.
+Graphical UIs will display the label when the Item is included, e.g. in [Basic UI]({{base}}/addons/uis/basic/readme.html) in a [Sitemap]({{base}}/configuration/sitemaps.html) definition.
 Some I/O services (e.g. the Amazon Alexa skill) also use the label to match an external voice command to an Item.
 
-In textual configurations the label, in quotation marks, appears next to the optional state presentation field in square brackets (see below).
-The label for the Item in the following example is "Temperature":
+In textual configurations the label in quotation marks, contains the label text and the optional state presentation field in square brackets (see below).
+They will be showed side by side, the text left aligned and the state representation right-aligned.
+The label text for the Item in the following example is "Temperature" and the optional state representation is set to be displayed e.g as "23.9 °C":
 
 ```java
 Number Livingroom_Temperature "Temperature [%.1f °C]"
 ```
 
+The label defined for an Item can be overwritten by the label definition on a sitemap.
+If the label definition on the Item contained label text and the state representation these have to be considered separatly.
+
+```perl
+sitemap demo label="My home automation" {
+    Frame label="Date" {
+        // Overrides label text only, keeps state representation defined by Item
+        Text item=Livingroom_Temperature label="Livingroom"
+        // Overrides label text and sets state representation to be empty, not displayed
+        Text item=Livingroom_Temperature label="Livingroom []"
+        // Overrides label text and state representation
+        Text item=Livingroom_Temperature label="Livingroom [%.2f °F]"
```

Comment:
I think the example is confusing. A user might expect automatic conversion...

Reply:
And i think he would get it if the temperature has an unit as defined for the UoM 

https://community.openhab.org/t/list-of-units-in-the-uom-unit-of-measurement/46842